### PR TITLE
Fixed #20415 -- Ensured srid isn't localized in OpenLayers JavaScript.

### PR DESCRIPTION
**Pull request by timgraham**
https://code.djangoproject.com/ticket/20415

(no test as in 0a29057ebea4335d503e373e4526b1c62c15bcdc)

### CODE REVIEW

File: django/contrib/gis/templates/gis/openlayers.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<style type="text/css">{% block map_css %}{% load i18n static %}{% get_current_language_bidi as LANGUAGE_BIDI %}
+{% load i18n]<style type="text/css">{% block map_css %}{% load i18n l10n static %}{% get_current_language_bidi as LANGUAGE_BIDI %}
```
Comment (charettes) [inline django/contrib/gis/templates/gis/openlayers.html]: I think there's something wrong with the `load` statement.

**Comment by charettes** [comment]
LGTM

**Comment by claudep** [comment]
Good, but a test would be nice (`tests/gis_tests/test_geoforms.py`?)

**Comment by timgraham** [comment]
test added

**Pull request merged**